src/Microsoft.AspNet.Mvc.Razor/TagHelpers/UrlResolutionTagHelper.cs

```diff
@@ -108,6 +107,11 @@ public override int Order
         /// <inheritdoc />
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
+            if (output == null)
```

Comment:
Seems like it. Maybe it's just less work to throw in the null checks here so that we don't have to add that check if `content` starts being used?